### ISSUE TITLE
[feature request] word meanings from google

### ISSUE THREAD

**Issue by omega3** [JunkenDuck/English-Definitions#1]
Can you consider this: as an option or a separate add-on.

I would like to have all word meanings in one line separated by slash, like this (output is in Polish):

zakończyć / wnioskować / wywnioskować / sfinalizować / konkludować / dokończyć / finalizować / zakończyć /

I would like to have it in a field like this (red letters):
https://i.imgur.com/hsZvTOX.png

Also Reverso lists meanings in similar way.

As reference:
translate-shell can do something similar, I mean create such list, so maybe some part of its code maybe useful
https://github.com/soimort/translate-shell/issues/368

Basically the author here is right, so if only possible to have slash instead of comma.
